game/world/opcode_handling/handlers/player/RandomRollHandler.py

```diff
@@ -16,7 +16,7 @@ def handle(world_session, socket, reader):
             roll_packet = PacketWriter.get_packet(OpCode.MSG_RANDOM_ROLL,
                                                   pack('<3IQ', minimum, maximum, roll, world_session.player_mgr.guid))
 
-            if world_session.player_mgr.group_manager:
+            if world_session.player_mgr.group_manager.is_party():
```

Comment:
shouldn't this check for `world_session.player_mgr.group_manager` being present first?

Reply:
Good catch.